packages/teams-components/src/strictStyles/mergeStrictClasses.ts

```diff
@@ -0,0 +1,19 @@
+import { mergeClasses } from '@fluentui/react-components';
+import type { MergeStrictClasses, StrictCssClass } from './types';
+import { createStrictClass } from './createStrictClass';
+import { validateStrictClasses } from './validateStrictClasses';
+
+export const mergeStrictClassesImpl: MergeStrictClasses = (
+  ...strictClasses: StrictCssClass[]
+) => {
+  strictClasses.forEach(validateStrictClasses);
+
+  const mergedClasses = mergeClasses(...strictClasses.map((x) => x.toString()));
+  return createStrictClass(mergedClasses);
+};
```

Comment:
- Let's use `.reduce()` to pass down falsy values
- It should be `(StrictCssClass | undefined)[]`

Reply:
I updated the signature to match `mergeClasses` to add `undefined | false`, not quite sure what you mean by using `reduce` though?